--- origsrc/gimp-ruby/Makefile.am	2007-12-18 00:28:05.000000000 -0600
+++ src/gimp-ruby/Makefile.am	2007-12-18 00:47:58.796875000 -0600
@@ -6,13 +6,6 @@
     plug-ins \
     po
 
-if PLATFORM_WIN32
-no_undefined = -no-undefined
-path_separator = ;
-else
-path_separator = :
-endif
-
 rubybase = ruby
 
 rubydir = $(GIMP_LIBDIR)/$(rubybase)
@@ -27,7 +20,7 @@
 
 install-env-files:
 	$(mkinstalldirs) '$(DESTDIR)$(rubyenvdir)'
-	echo '$(path_separator) RUBYLIB=$${gimp_plug_in_dir}/$(rubybase)' > '$(DESTDIR)$(rubyenvfile)'
+	echo '$(PATHSEP) RUBYLIB=$${gimp_plug_in_dir}/$(rubybase)' > '$(DESTDIR)$(rubyenvfile)'
 	cp 'ext/$(rubyconsolefile)' '$(DESTDIR)$(rubydir)'
 
 install-interp-file:
--- origsrc/gimp-ruby/configure.in	2007-12-18 00:28:05.000000000 -0600
+++ src/gimp-ruby/configure.in	2007-12-18 00:28:59.765625000 -0600
@@ -353,7 +353,7 @@
 AC_MSG_CHECKING([whether symbols are prefixed])
 
 case "$host_os" in
-  mingw*)
+  mingw* | cygwin*)
     SYMPREFIX='_'
     ;;
   darwin* | rhapsody* | machten*)
--- origsrc/gimp-ruby/ext/Makefile.am	2007-12-18 00:28:04.000000000 -0600
+++ src/gimp-ruby/ext/Makefile.am	2007-12-18 00:30:43.312500000 -0600
@@ -2,9 +2,6 @@
 
 if PLATFORM_WIN32
 no_undefined = -no-undefined
-path_separator = ;
-else
-path_separator = :
 endif
 
 libdir = $(GIMP_LIBDIR)/ruby
